tests/addresslib/parser_address_list_test.py

```diff
@@ -210,64 +191,4 @@ def test_delimiters_relaxed():
             continue
 
         addr_string = 'bill@microsoft.com' + ''.join(e) + 'steve@apple.com, torvalds@kernel.org'
-        run_relaxed_test(addr_string, [BILL_AS, STEVE_AS, LINUS_AS], [])
-
-def test_delimiters_strict():
-    # permutations
-    for e in permutations('  ,,;;'):
-        addr_string = 'bill@microsoft.com' + ''.join(e) + 'steve@apple.com, torvalds@kernel.org'
-        run_strict_test(addr_string, [BILL_AS, STEVE_AS, LINUS_AS])
-
-    # powerset
-    for e in powerset('  ,,;;'):
-        # empty sets will be tested by the synchronize tests
-        if ''.join(e).strip() == '':
-            continue
-
-        addr_string = 'bill@microsoft.com' + ''.join(e) + 'steve@apple.com, torvalds@kernel.org'
-        run_strict_test(addr_string, [BILL_AS, STEVE_AS, LINUS_AS])
-
-
-def test_synchronize_relaxed():
```

Comment:
Since the parser has been completely rewritten the parser synchronization works differently now.